@3rdweb-sdk/react/hooks/useApi.ts

```diff
@@ -441,6 +442,11 @@ export function useUpdateAccountPlan() {
         throw new Error(json.error.message);
       }
 
+      // Wait for account plan to update via stripe webhook
+      if (waitForWebhook) {
+        await new Promise((resolve) => setTimeout(resolve, 1000 * 10));
+      }
```

Comment:
just a note, not a blocker, having to do something like this here in the UI means we've incorrectly designed the data flow, cc @edwardysun

Reply:
Yeah I agree, this project has a pretty tight timeline so we weren't able to go in and redesign the data flows but we should reconsider how we're storing the billing data at some point (not just sitting on the account)